tests/fmt.toml

```diff
@@ -1,5 +1,4 @@
 comment_width=100
-wrap_comments=true
```

Comment:
I had incorporated some integ testing, for this PR and then removed it. I don't recall changing this though. This may be a rebase artefact, I will undo this.